Makefile

```diff
@@ -15,8 +15,11 @@ client-python:
 build:
 	hack/dockerized "./hack/check.sh && KUBEVIRT_VERSION=${KUBEVIRT_VERSION} ./hack/build-go.sh install ${WHAT}" && ./hack/build-copy-artifacts.sh ${WHAT}
 
+coverage:
+	hack/dockerized "./hack/check.sh && ./hack/coverage.sh ${WHAT}"
```

Comment:
It seems like you did not add a `hack/coverage.sh` file. Maybe you forgot to add it?

Reply:
Forgot to add it, now is there, thanks to point it out!!